### PR TITLE
Typograpghy

### PR DESCRIPTION
**Pull request by AmrutaJayanti**
## Description

Added Typography.md

###Issues Related:

Closes: #316 

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [x] Other (specify): Adding Solution

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: docs/CSS/css-basics/Typography.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    font-size: 16px;
+}
+
+h1 {
+    font-size: 2em; /* 2 times the current font size */
+}
+```
+You can check the relation between units
+
+<img src="https://tse4.mm.bing.net/th?id=OIP.x3eQ0WknsCLHiRLl5jGYowHaEr&pid=Api&P=0&h=180" alt="CSS" height= "300" width="500"/>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-basics/Typography.md:30]: ## Use Table and remove `img`

For Example Table in Markdown

```md
Here is an example of a table in Markdown:

| Header 1 | Header 2 | Header 3 |
|----------|----------|----------|
| Row 1, Col 1 | Row 1, Col 2 | Row 1, Col 3 |
| Row 2, Col 1 | Row 2, Col 2 | Row 2, Col 3 |
| Row 3, Col 1 | Row 3, Col 2 | Row 3, Col 3 |

You can add alignment to the columns as well:

| Left-Aligned | Center-Aligned | Right-Aligned |
|:-------------|:--------------:|--------------:|
| This         | This           | This          |
| column       | column         | column        |
| will         | will           | will          |
| be           | be             | be            |
| left-aligned | center-aligned | right-aligned |
```

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
you need improve your content more valuable like our format and change some suggestions.

**Comment by AmrutaJayanti** [comment]
@Ajay-Dhangar Thank you ! I made the changes . Please approve

**Review by Ajay-Dhangar — APPROVED** [review]
DOne

**Pull request merged**